README.md

```diff
@@ -41,11 +41,11 @@ print(y.grad)  # dz/dy
 
 It turns out, a decent autograd tensor library is 90% of what you need for neural networks. Add an optimizer (SGD and Adam implemented) from tinygrad.optim, write some boilerplate minibatching code, and you have all you need.
 
-### Neural network example (from test/mnist.py)
+### Neural network example (from test/test_mnist.py)
 
 ```python
 from tinygrad.tensor import Tensor
-import tinygrad.optim as optim
```

Comment:
I like to import as optim, it matches torch.

Reply:
Done